moveit_ros/benchmarks/scripts/moveit_benchmark_statistics.py

```diff
@@ -318,7 +318,10 @@ def plotAttribute(cur, planners, attribute, typename):
         measurementsPercentage = [sum(m) * 100. / len(m) for m in measurements]
         ind = range(len(measurements))
         plt.bar(ind, measurementsPercentage, width)
-        xtickNames = plt.xticks([x + width / 2. for x in ind], labels, rotation=30, fontsize=8)
+	### Insert take out of this term, a practical solution for OMPL solvers
+	labels = [l.replace('kConfigDefault', '') for l in labels]
```

Comment:
Sorry, but although I agree this might look cleaner, I believe the name needs to keep it's full length, as it reflects the configuration specified in `config/ompl_planning.yaml`. For most planners, this involves a number of parameters and you can specify arbitrary names in the config referring to the same planner with different parameterization.
Please remove this again for upstream.

Reply:
I will comment this line and specify the reason above, is this ok to you? Or I simply remove?

Reply:
I made your changes in the second commit, please check.
Thanks,